cairo/tests/test_serde.py

```diff
@@ -264,14 +264,17 @@ def test_type(
             MessageCallOutput,
             Union[Bytes, LegacyTransaction],
             Union[Bytes, Receipt],
+            Union[Bytes, Withdrawal],
             Optional[Union[Bytes, LegacyTransaction]],
             Optional[Union[Bytes, Receipt]],
+            Optional[Union[Bytes, Withdrawal]],
```

Comment:
missing `Mapping[Bytes, Optional[Union[Bytes, Withdrawal]]]`
 To apply to similar types (receipt, legacyTx)

Reply:
it doesn't really matter as long as the full type is present (Trie[Bytes, Optional[...]])

Reply:
just for info, it doesn't really matter as it's covered by the super-type in Trie[...]